tensorflow/python/ops/array_ops.py

```diff
@@ -1152,6 +1152,37 @@ def matrix_transpose(a, name="matrix_transpose"):
 # pylint: enable=invalid-name
 
 
+def strings(shape, dtype=dtypes.string, name=None):
+  """Create a tensor with all elements set to ''(empty string)
+  
+  This operation returns a tensor of type `dtype` with shape `shape` and
+  all elements set to ''(empty string).
+  
+  For example:
+
+  ```python
+  tf.strings([2,3], tf.string) ==> [['', '', ''], ['', '', '']]
+  ```
+  Args:
+    shape: Either a list of integers, or a 1-D `Tensor` of type `string`.
+    dtype: The type of an element in the resulting `Tensor`.
+    name: A name for the operation (optional).
+
+  Returns:
+    A `Tensor` with all elements set to ''(empty string)
+  """
+  with ops.name_scope(name, 'strings', [shape]) as name:
+    string = ''
+    try:
```

Comment:
Yep, you're right~ This function is just for `tf.string`, so it doesn't need a dtype argument